usercp.php

```diff
@@ -2102,7 +2102,7 @@
 	}
 }
 
-if($mybb->input['action'] == "removesubscription")
+if($mybb->input['action'] == "removesubscription" && ($mybb->request_method == "post" || verify_post_check($mybb->get_input('my_post_key'), true)))
```

Comment:
Is that to allow old links (GET) with `my_post_key` to work?

Reply:
Yes. The "remove forum subscription" link in forumdisplay still uses this.